assets/lang/sklmessages_pl_PL.properties

```diff
@@ -83,21 +83,21 @@ settings.title=Ustawienia launchera
 settings.lang=Język
 settings.lang.help=Widzisz błąd w tłumaczeniu?
 settings.lang.link=Pomóż go naprawić!
-settings.theme=Motyw
+settings.theme=Motywy
 settings.theme.light=Jasny motyw
 settings.theme.dark=Ciemny motyw
 settings.token=Specjalny klucz dostępu
 settings.alert.title=Informacja
-settings.alert.text=Musisz zrestartować launcher, aby zastosować zmiany.
+settings.alert.text=Musisz uruchomić ponownie launcher, aby zastosować zmiany.
 settings.cancel=Anuluj
 settings.save=Zapisz
 
-crash.sorry=Ups! Wygląda na to, że gra się scrashowała. Przepraszamy za niedogodność :(
+crash.sorry=Ups! Wygląda na to, że gra się scrashowała. Przepraszamy za niedogodności :(
```

Comment:
Tutaj również na plus. Teraz zdanie to brzmi lepiej niż poprzednio, w j. polskim oczywiście. Nie wiem natomiast, czy zmiana na liczbę mnogą również w pliku źródłowym będzie także na plus. Może warto to rozważyć.